spatialprofilingtoolbox/workflow/assets/.workflow.config.template

```diff
@@ -1,8 +1,9 @@
 [general]
 db_config_file = path/to/db.config
 executor = local
-sif_file = path/to/sif/file
 excluded_host = None
+orchestrator = local
```

Comment:
I think "orchestrator" may be a bit of a misnomer here.
Nextflow is the orchestrator. The options for this field are "local", "docker", or "singularity", and these all describe a type of platform that the orchestrator calls upon for running individual processes.
Perhaps "process_run_delegation"?

Reply:
This is fair. I struggled to disambiguate between the "executor" and the "orchestrator," and per your comments, Docker/Singularity/local is yet another distinct concept. Maybe something more apt would be [a "container" platform, as SingularityCE describes itself in its docs.](https://docs.sylabs.io/guides/latest/user-guide/introduction.html)

Reply:
As discussed, how about: `container_platform` as `docker`, `singularity`, or `none`, with default value `none`.